app/uploaders/supporting_evidence_attachment_uploader.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 # Files uploaded as part of a form526 submission that will be sent to EVSS upon form submission.
+# [wipn8923] update?
```

Comment:
https://github.com/department-of-veterans-affairs/vets-api/pull/13835/commits/7e5c4658bb3877539f4f9fcad82cae95f5421bcd